### DISCUSSION TITLE
Can't exclude some language

### DISCUSSION THREAD

**Discussion by SamLam140330** [github-linguist/linguist#5193] [Q&A] [answered]
I am learning Unity to design a game and I am using C# as a programming language. However, it shows the other two languages that I didn't write.
![1](https://user-images.githubusercontent.com/48656764/107114246-5334f380-689f-11eb-8e38-d9ca9224860a.png)
Is it possible to exclude those two languages?
![2](https://user-images.githubusercontent.com/48656764/107114251-562fe400-689f-11eb-9d6a-803dbbd0c633.png)
 I have tried to make a .gitattributes but it seems not working. Is my .gitattributes wrong?
![3](https://user-images.githubusercontent.com/48656764/107114304-ce96a500-689f-11eb-8127-e18c58b3e594.png)
Thank you for your help :)

**Comment by lildude** [comment, marked as answer]
>  Is my .gitattributes wrong?

Yup... two things:

- your path name is wrong... you're missing an `s`, ie `shader` should be `shaders`
- your want to mark the directories as vendored so you want to use either `linguist-vendored` or `linguist-vendored=true`, ie the opposite of what you're doing.

**Reply by SamLam140330** [reply]
Oh, got it
Thank you so much

**Comment by AkashAfzal** [comment]
use these two lines

```
*.shader linguist-detectable=false
*.cginc linguist-detectable=false
```
